curriculum/challenges/_meta/lab-recipe-page/meta.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "Recipe Page",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "dashedName": "lab-recipe-page",
+  "order": 2,
```

this should be 3 according to the sheet

```suggestion
  "order": 3,
```

---

We will have to adjust the order later I think. It breaks if we are missing an order number - e.g. we can't have blocks with order 0,1, and 3 cause 2 is missing. I can maybe look into changing that so we don't have to go back and make those adjustments, but it's a safeguard against typos and unintended ordering.